system-addon/lib/SnippetsFeed.jsm

```diff
@@ -27,12 +33,22 @@ this.SnippetsFeed = class SnippetsFeed {
   constructor() {
     this._refresh = this._refresh.bind(this);
   }
+
   get snippetsURL() {
     const updateURL = Services
       .prefs.getStringPref(SNIPPETS_URL_PREF)
       .replace("%STARTPAGE_VERSION%", STARTPAGE_VERSION);
     return Services.urlFormatter.formatURL(updateURL);
   }
+
+  isDefaultBrowser() {
+    try {
+      return ShellService.isDefaultBrowser();
+    } catch (e) {}
+    // istanbul ignore next
+    return null;
```

Comment:
nit: indentation is off i think. also, do we want to return false for consistency or is it necessary to return null for some other part of the code to know it went wrong?